kube/src/api/typed.rs

```diff
@@ -108,19 +108,19 @@ where
     /// Create a resource
     ///
     /// This function requires a type that Serializes to `K`, which can be:
-    /// 1. Raw string yaml
-    ///   - easy to port from existing files
-    ///   - error prone (run-time errors on typos due to failed serialize attempts)
-    ///   - very error prone (can write invalid yaml)
+    /// 1. Raw string YAML
+    ///     - easy to port from existing files
+    ///     - error prone (run-time errors on typos due to failed serialize attempts)
+    ///     - very error prone (can write invalid YAML)
     /// 2. An instance of the struct itself
-    ///   - easy to instantiate for CRDs (you define the struct)
-    ///   - dense to instantiate for k8s-openapi types (due to many optionals)
-    ///   - compile-time safety
-    ///   - but still possible to write invalid native types (validation at apiserver)
-    /// 3. `serde_json::json!` macro instantiated `serde_json::Value`
-    ///   - Tradeoff between the two
-    ///   - Easy partially filling of native k8s-openapi types (most fields optional)
-    ///   - Partial safety against runtime errors (at least you must write valid json)
+    ///     - easy to instantiate for CRDs (you define the struct)
+    ///     - dense to instantiate for [`k8s_openapi`] types (due to many optionals)
+    ///     - compile-time safety
+    ///     - but still possible to write invalid native types (validation at apiserver)
+    /// 3. [`serde_json::json!`] macro instantiated [`serde_json::Value`]
+    ///     - Tradeoff between the two
+    ///     - Easy partially filling of native [`k8s_openapi`] types (most fields optional)
+    ///     - Partial safety against runtime errors (at least you must write valid JSON)
```

Comment:
The list wasn't rendered correctly:

<img src="https://user-images.githubusercontent.com/639336/103120947-be4ba300-462e-11eb-965c-10a70bc7918b.png" width=400>

<img src="https://user-images.githubusercontent.com/639336/103120951-c572b100-462e-11eb-894d-cce77bab7452.png" width=400>

Reply:
Ah. I for some reason I never write `yaml` capitalised. Dunno why, it feels screaming whilst undeserving/unworthy of highlighting and capitalisation.

But yours is probably what most people expect though.

Reply:
I'll revert it if you'd like. I mainly wanted to fix the list.

Reply:
NP. Consistency with language is more important :-)